.gitignore

```diff
@@ -1,8 +1,58 @@
+
+# editor related
+
+.idea/
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
 *.pyc
 *~*
 *#*
 py3env
 dist
 build
 *.egg*
+*.com
+*.class
+*.dll
+*.exe
+*.o
+*.so
+
+# temporary files
+.DS_Store
+*.swp
+*.swo
+*.json
+*.conf
+
+# Packages
+
+# it's better to unpack these files and commit the raw source
+# git has its own built in compression methods
+*.7z
+*.dmg
+*.gz
+*.iso
+*.jar
+*.rar
+*.tar
+*.zip
+
+# Logs and databases
+
+*.log
+*.sql
+*.sqlite
+
+# OS generated files #
 
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
```

Comment:
Please add a line feed to the bottom of this file.

Reply:
@bobinson - :)

Reply:
sure - I had removed that to "save space" ;-) - I had done some programming with PIC microprocessors for a coffee vending machine in my first job and that made me too conscious about "space"